packages/typegen/src/util.ts

```diff
@@ -78,3 +82,33 @@ export const changedFiles = (params: {since: string; cwd: string}) =>
     .filter(Boolean)
 
 export const globList = (list: readonly string[]) => (list.length === 1 ? list[0] : `{${list.join(',')}}`)
+
+export const tsCustom = (() => {
+  const isSqlIdentifier = (node: ts.Node) => ts.isIdentifier(node) && node.getText() === 'sql'
+  const isSqlPropertyAccessor = (e: ts.Expression) => ts.isPropertyAccessExpression(e) && isSqlIdentifier(e.name)
+  /**
+   * Checks if the supplied node is an sql template literal
+   */
+  const isSqlLiteral = (node: ts.Node): node is ts.TaggedTemplateExpression =>
+    ts.isTaggedTemplateExpression(node) && (isSqlIdentifier(node.tag) || isSqlPropertyAccessor(node.tag))
+  return {
+    isSqlLiteral,
+  } as const
```

Comment:
minor: what's the advantage of `as const` here?

Reply:
Ok, sure. I generally don’t use readonly modifiers - I just don’t mutate objects but no harm in this.